zerver/views/upload.py

```diff
@@ -224,9 +241,52 @@ def serve_image_error(status: int, image_path: str) -> HttpResponseBase:
         patch_vary_headers(response, ("Accept",))
         return response
     if url_only:
+        # TODO: Does this need to handle thumbnails?
```

Comment:
I think this doesn't; the feature is intended for PDFs and the like: 0ccc0f02ce6d20cbf59b86b7073d6124c2acdb2b

Reply:
https://chat.zulip.org/#narrow/stream/378-api-design/topic/thumbnailing.20URLs/near/1891180